ms/MSSel/MSAntennaParse.h

```diff
@@ -174,6 +174,7 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
   public:
     static MSAntennaParse* thisMSAParser;
     static MSSelectionErrorHandler* thisMSAErrorHandler;
+    static void cleanupErrorHandler() {if (thisMSAErrorHandler) delete thisMSAErrorHandler;thisMSAErrorHandler=0x0;}
```

Comment:
Makes debugging easier -- for me anyway.

Is there are document I can read about coding guidelines for CASACore now?
I suppose (and hope) code reviews will also follow that.  I want to avoid
protracted email discussions which beyond a point have to wander into the
design (and IMHO this is not the place for that discussion).

On Wed, Sep 30, 2015 at 9:54 AM, Julian Taylor <notifications@github.com>
wrote:

> In ms/MSSel/MSAntennaParse.h
> <https://github.com/casacore/casacore/pull/252#discussion_r40812295>:
>
> > @@ -174,6 +174,7 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
> >    public:
> >      static MSAntennaParse* thisMSAParser;
> >      static MSSelectionErrorHandler* thisMSAErrorHandler;
> > +    static void cleanupErrorHandler() {if (thisMSAErrorHandler) delete thisMSAErrorHandler;thisMSAErrorHandler=0x0;}
>
> why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/casacore/casacore/pull/252/files#r40812295>.[image:
> Web Bug from
> https://github.com/notifications/beacon/AM3Q7fh0nnJPns3SuNdRS67B0Cq69yP_ks5o2_03gaJpZM4GGG24.gif]
>



-- 
संजय